### DISCUSSION TITLE
SONOFF 4CH PRO R3 - TASMOTA 9.3.1 FERNBEDIENUNG ANLERNEN

### DISCUSSION THREAD

**Discussion by Frabefe** [arendst/Tasmota#11806] [Q&A] [answered]
Hallo Leute,

habe mir den Sonoff 4CH Pro R3 zugelegt und mit Tasmota 9.3.1 geflasht, funktioniert auch soweit wie er soll!

Jetzt würde ich gerne auch die Möglichkeit der Fernbedienung mit RF 433 MHz war nehmen, ich habe eine 4 Tasten Fernbedienung und die würde ich gerne anlernen. 

Habe schon mehrmals mit zweimal Taste drücken versucht, aber da wird nur das Relay geschaltet, sonst passiert nichts.

Über euere Hilfe würde ich mich sehr freuen!

Vielen Dank vorab.

LG Paco

**Comment by joba-1** [comment]
kann mich leider nicht mehr so genau dran erinnern wie das ging, aber es hatte mit rfkey Befehlen auf der Console zu tun. Da findet sich bestimmt was dazu in der Doku...

**Reply by Frabefe** [reply]
Danke,
habe es auch schon über der Console mit rfkey probiert {"Command":"Unknown"}
leider

**Reply by joba-1** [reply]
war das nicht rfkeyX (X=1,2,...)? Aber ohne die Doku zu lesen wird dir das auch nicht helfen, sorry

**Comment by Jason2866** [comment, marked as answer]
Hallo,
grundsätzlich ist hier Englisch gewünscht um allen User in der Welt die Infos zugänglich zu machen.
Sonoff 4ch pro hat keinen rf empfänger der die empfangenen codes "rausgibt".
Das anlernen funktioniert hardwarebasiert außerhalb des ESP82XX! und funktuoniert unverändert (oder eben auch nicht) wie mit der orig. Firmware.
